homeassistant/components/risco/binary_sensor.py

```diff
@@ -3,13 +3,25 @@
     DEVICE_CLASS_MOTION,
     BinarySensorEntity,
 )
+from homeassistant.helpers import config_validation as cv, entity_platform
 
 from .const import DATA_COORDINATOR, DOMAIN
 from .entity import RiscoEntity
 
+SERVICE_BYPASS_ZONE = "bypass_zone"
+SERVICE_UNBYPASS_ZONE = "unbypass_zone"
+
 
 async def async_setup_entry(hass, config_entry, async_add_entities):
     """Set up the Risco alarm control panel."""
+    platform = entity_platform.current_platform.get()
+    platform.async_register_entity_service(
+        SERVICE_BYPASS_ZONE, cv.make_entity_service_schema({}), "async_bypass_zone"
```

Comment:
```suggestion
        SERVICE_BYPASS_ZONE, {}, "async_bypass_zone"
```

Reply:
Cool, I didn't know it's handled automatically.